packages/graphql/src/index.ts

```diff
@@ -41,17 +40,13 @@ export type GraphQLClientQuery = {
  * ```
  */
 export const createGraphQLClient =
-  (
-    url: MaybeAccessor<string>,
-    headers?: RequestHeaders,
-    fetchFn?: typeof fetch
-  ): GraphQLClientQuery =>
+  (url: MaybeAccessor<string>, options?: RequestOptions): GraphQLClientQuery =>
```

Comment:
I that passing an entire options obj is fine, but `variables` come from a different place.
```suggestion
  (url: MaybeAccessor<string>, options?: Omit<RequestOptions, "variables">): GraphQLClientQuery =>
```